src/core/main.c

```diff
@@ -1647,6 +1647,7 @@ static int invoke_main_loop(
                                 log_warning_errno(r, "Failed to parse config file, ignoring: %m");
 
                         set_manager_defaults(m);
+                        set_manager_settings(m);
```

Comment:
hmmpf, so these settings overlap with what can be specified on the kernel cmdline (and also with settings set via env vars). Usually the kernel cmdline stuff overrides the configuration files stuff. If we'd load these settings here too, then behaviour at boot would be different from after the first reload.

The stuff in set_manager_defaults() is different, as it cannot be overriden by kernel cmdline stuff generally. I mean that#s one of the reasons these calls are separate...

i mean, we could look into cleaning this up, and implement the full merging from other config sources here, but that should happen in a separate PR and is probably going to be quite some work. Hence, please let's drop this commit from this PR for now

Reply:
so you asked me to move log level/target settings in set_manager_settings() but this function shouldn't be called in the reload path... I'm wondering how this is going to work then.